kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -182,6 +181,8 @@ fun compareDouble(path: List<String>, expected: JsonNode.DoubleNode, actual: Jso
    return when {
       actual is JsonNode.DoubleNode -> compareDoubles(path, expected.value, actual.value)
       actual is JsonNode.LongNode -> compareDoubles(path, expected.value, actual.value.toDouble())
+      actual is JsonNode.FloatNode -> compareDoubles(path, expected.value, actual.value.toDouble())
```

Comment:
Is float actually used if we're just parsing strings? Like how does it know that 0.2 should be a float and not a double ?

Reply:
So for just json strings the answer is no. I guess we have two options here: remove the case here since it appears to always actually parse as a double, or keep the float option to eventually expand this assertions library to include the other serialization kinds that kotlinx-serialization supports.

Reply:
I guess if they provide a float node as part of their ADT then it won't hurt to include it here.

Reply:
👍 sounds good to me